recipe/meta.yaml

```
@@ -24,78 +24,76 @@ requirements:
     - python-build
     - meson-python ==0.13.1
     - meson ==1.2.1
-    - cython >=0.29.33,<3
+    - cython >=3.0.5
     - numpy {{ numpy }}
     - versioneer
-    - tomli # [py<311]
     - pip
   run:
     - python
     # Base Dependencies
     - {{ pin_compatible('numpy') }}
     - python-dateutil >=2.8.2
     - pytz >=2020.1
-    - python-tzdata >=2022.1
+    - python-tzdata >=2022.7
     # Recommended dependencies to achieve large speedups
     # see https://pandas.pydata.org/pandas-docs/version/2.1.1/getting_started/install.html#performance-dependencies-recommended
-    - numexpr >=2.8.0
-    - bottleneck >=1.3.4
+    - numexpr >=2.8.4
+    - bottleneck >=1.3.6
   run_constrained:
     # Optional dependencies
     # see https://pandas.pydata.org/pandas-docs/version/2.1.1/getting_started/install.html#optional-dependencies
     # and https://github.com/pandas-dev/pandas/blob/v2.1.1/pyproject.toml
     # performance
-    - numba >=0.55.2
+    - numba >=0.56.4
     # plot
-    - matplotlib-base >=3.6.1
+    - matplotlib-base >=3.6.3
     # output_formatting
     - jinja2 >=3.1.2
-    - tabulate >=0.8.10
+    - tabulate >=0.9.0
     # computation
-    - scipy >=1.8.1
-    - xarray >=2022.03.0
+    - scipy >=1.10.0
+    - xarray >=2022.10.0
     # fss
-    - fsspec >=2022.05.0
+    - fsspec >=2022.11.0
     # aws
-    - s3fs >=2022.05.0
+    - s3fs >=2022.11.0
     # gcp
-    - gcsfs >=2022.05.0
-    - pandas-gbq >=0.17.5
+    - gcsfs >=2022.11.0
+    - pandas-gbq >=0.19.0
     # excel
     - odfpy>=1.4.1
-    - openpyxl >=3.0.10
-    - pyxlsb >=1.0.9
+    - openpyxl >=3.1.0
+    - pyxlsb >=1.0.10
     - xlrd >=2.0.1
-    - xlsxwriter >=3.0.3
+    - xlsxwriter >=3.0.5
     # parquet, feather
-    - pyarrow >=7.0.0
+    - pyarrow >=10.0.1
     # hdf5
-    - blosc >=1.21.0
-    - pytables >=3.7.0
+    - blosc >=1.20.1
+    - pytables >=3.8.0
     # spss
-    - pyreadstat >=1.1.5
+    - pyreadstat >=1.2.0
     # DB
-    - sqlalchemy >=1.4.36
-    - psycopg2 >=2.9.3
+    - sqlalchemy >=2.0.0
+    - psycopg2 >=2.9.6
     - pymysql >=1.0.2
     # html, xml
-    - beautifulsoup4 >=4.11.1
+    - beautifulsoup4 >=4.11.2
     - html5lib >=1.1
-    - lxml >=4.8.0
+    - lxml >=4.9.2
     # compression
-    - zstandard >=0.17.0
+    - zstandard >=0.19.0
     # clipboard
-    - pyqt >=5.15.6,<6
-    - qtpy >=2.2.0
+    - pyqt >=5.15.9
+    - qtpy >=2.3.0
     # consortium-standard
     - dataframe-api-compat >=0.1.7
     # uncategorized
-    - fastparquet >=0.8.1
+    - fastparquet >=2022.12.0
```

```suggestion
    - fastparquet >=2022.12.0
    # testing
    - hypothesis >=6.46.1
    - pytest >=7.3.2
    - pytest-xdist >=2.2.0
```
I know these are typically used for tests, but pandas lets you do testing, so I think we should also add these as dependencies, but please educate me if I'm wrong.

---

They are in the requires section of the test portion already

---

Yeah, but I believe they can actually be used during runtime as well, so I'm not sure if they should not be added here just because they are already in the test section below. Maybe I'm misunderstanding the difference between adding a dependency to the test section as opposed to adding it to the run section and the risks of having them listed in both sections versus one or the other.

---

`hypothesis`, `pytest` and `pytest-xdist` are only used in pandas' test suite.

---

Oh okay. I thought they were used as a part of one of their features, which is to do testing, as well as used in their test suite. My mistake!